### ISSUE TITLE
Lack support for c-style comment for the proto file, which official compiler C++/Java/Python do support

### ISSUE THREAD

**Issue by lulyon** [allegro/php-protobuf#69]
With protocol file `Hello.proto`
```
/*
** Hello **
*/
message Hello {
    optional string world = 1;
}
```

>`>` php protoc-php.php Hello.proto
Proto file missformed

While official protocol compiler for C++/Java/Python works fine with this protocol file. Would this feature be implemented soon?

**Comment by hjagodzinski** [comment]
I have already mentioned it it your other issue report but for clarity: yes, I'm working on using the official protoc compiler.

**Comment by hjagodzinski** [comment]
Fixed in #74.
